apps/authorization/tests/test_data_access_grant_permissions.py

```diff
@@ -435,17 +434,17 @@ def test_research_study_app_type_without_switch_limit_data_access(self):
             expected_response_code=200,
         )
 
-    @override_switch("limit_data_access", active=True)
-    def test_research_study_app_type_with_switch_limit_data_access(self):
+    @override_flag("limit_data_access", active=True)
+    def test_research_study_app_type_with_flag_limit_data_access(self):
         """
         Test Application.data_access_type="RESEARCH_STUDY".
 
         Test data access for FHIR and profile end points
-        with limit_data_access switch True.
+        with limit_data_access flag True.
 
-        This will be the switch setting in PROD.
+        This will be the flag setting in SBX.
```

Comment:
I think this be PROD instead of SBX in comment?

Reply:
Yep.  Meant to do that.